app/design/frontend/UnitedArtsEducation/unitednow/Magento_Checkout/web/js/view/configure/product-customer-data.js

```diff
@@ -0,0 +1,68 @@
+/**
```

Comment:
Is it a requirement to use that logic?

Reply:
Yes, otherwise the cart-edit functionality is broken in IE11. (When the user tried to edit the quantity of an item in the cart, the quantity box was empty, instead of showing the quantity in the cart, like it was supposed to.)